src/views/WordDetail.vue

```diff
@@ -67,6 +67,84 @@
           >
             {{ data.description }}
           </div>
+          <h3>{{ $t('views.WordDetail.BondingCurve') }}</h3>
+          <div
+            v-if="data"
+            class="asset-details__chart"
+          >
+            <figure>
+              <BondingCurve :data="data" />
+              <figcaption class="legend">
+                <div class="legend-item">
+                  <h3>{{ $t('views.WordDetail.LegendBuyPrice') }}</h3>
+                  <div>
+                    <AeAmount
+                      :amount="data.buyPrice"
+                      aettos
+                    />
+                    <FiatValue
+                      :amount="data.buyPrice"
+                      aettos
+                    />
+                  </div>
+                </div>
+                <div class="legend-item">
+                  <h3>{{ $t('views.WordDetail.LegendSellPrice') }}</h3>
+                  <div>
+                    <AeAmount
+                      :amount="data.sellPrice"
+                      aettos
+                    />
+                    <FiatValue
+                      :amount="data.sellPrice"
+                      aettos
+                    />
+                  </div>
+                </div>
+                <div class="legend-item">
+                  <h3>{{ $t('views.WordDetail.LegendSupply') }}</h3>
+                  <div>
+                    <AeAmount
+                      :token="data.tokenAddress"
+                      :amount="data.totalSupply"
+                      aettos
+                    />
+                  </div>
+                </div>
+                <div class="legend-item">
+                  <h3>{{ $t('views.WordDetail.LegendInitialPrice') }}</h3>
+                  <div>
+                    <AeAmount
+                      :amount="0"
+                      aettos
+                    />
+                    <FiatValue
+                      :amount="0"
+                      aettos
+                    />
+                  </div>
+                </div>
+                <div class="legend-item">
+                  <h3>{{ $t('views.WordDetail.LegendSpread') }}</h3>
+                  <div>
+                    <AeAmount
+                      :amount="data.spread"
+                      aettos
+                    />
+                  </div>
+                </div>
+                <div class="legend-item">
+                  <h3>{{ $t('views.WordDetail.LegendReserve') }}</h3>
+                  <div>
+                    <AeAmount
+                      :amount="data.sellPrice * data.totalSupply / 2000000000000000000"
```

Comment:
whats that devision by? no magic numbers please

Reply:
Actually not sure how "Reserve" should be calculated, I assumed it is the area,

Reply:
The reserve is everything below the sell curve.